rdrf/rdrf/patients_listing.py

```diff
@@ -84,8 +84,7 @@ def get_columns(self):
         return [
             ColumnFullName(_("Patient"), "patients.can_see_full_name"),
             ColumnDateOfBirth(_("Date of Birth"), "patients.can_see_dob"),
-            ColumnPatientType(_("Type"), "patients.can_see_type"),
-            ColumnPatientGender(_("Gender"), "patients.can_see_gender"),
+            ColumnCodeField(_("Code Field"), "patients.can_see_code_field"),
```

Comment:
This is actually a custom permission name - We probably haven't created a custom permission object for this yet 
I'll fix up in my branch though